framework/src/bcs/ArrayLowerDIntegratedBC.C

```diff
@@ -159,7 +154,13 @@ ArrayLowerDIntegratedBC::computeLowerDOffDiagJacobian(Moose::ConstraintJacobianT
 
   const auto & jvar = getVariable(jvar_num);
 
-  prepareMatrixTagLower(_assembly, ivar, jvar_num, type);
+  // need to transform the type for assembling Jacobian on boundary to be consistent with
+  // Assembly::addJacobianLowerD() and Assembly::prepareLowerD().
+  Moose::ConstraintJacobianType type_tr =
```

Comment:
spell out `tr`. I assume it means transform, but I'm not 100% sure. Code should be self-documenting as much as possible

Reply:
yes, tr here means transformed. We had inline comment above, but I can rename it.

Reply:
Even though you have inline comments, it's still good practice to have meaningful variable names. I was able to guess that `tr` meant transform from your inline comments so I appreciate them, but I'd prefer not to guess. `tr` is a method name on tensors for example that signifies trace. I just don't want to have to guess